usermods/PIR_sensor_switch/usermod_PIR_sensor_switch.h

```diff
@@ -272,7 +271,7 @@ void PIRsensorSwitch::publishMqtt(const char* state)
   //Check if MQTT Connected, otherwise it will crash the 8266
   if (WLED_MQTT_CONNECTED) {
     char buf[64];
-    sprintf_P(buf, PSTR("%s/motion"), mqttDeviceTopic);   //max length: 33 + 7 = 40
+    sprintf_P(buf, PSTR("%s/motion_%s"), mqttDeviceTopic,_name);   //max length: 33 + 7 = 40

```

Comment:
This will break existing integrations (mine included). Unacceptable change,

Reply:
I am not familiar with why this breaks it , although I will find a better solution for a better integration .